src/components/Select.vue

```diff
@@ -216,7 +216,6 @@
   }
   .v-select input[type="search"].hidden {
     border: none;
-    height: 0;
```

Comment:
I seem to recall adding this for a reason... but now I can't recall what it was. In any case, I don't understand why you'd not be able to click to trigger the dropdown. The actual `input` is already `width: 0` in the case of it being hidden. Hmm....